Examples/Algorithms/Alignment/include/ActsExamples/Alignment/AlignmentAlgorithm.hpp

```diff
@@ -24,6 +25,35 @@
 
 namespace ActsExamples {
 
+class AlignmentGroup {
```

Comment:
Ultimately this should go into core. I don't think it has any Examples dependency, so I think it should be an easy move.

Reply:
We can leave it like this for this PR. I don't think there is an alignment plugin at this point, and I'd probably advocate making this a subdirectory in Core directly.

Reply:
Sounds good to me!